plugin-scalebar/src/main/java/com/mapbox/maps/plugin/scalebar/ScaleBarImpl.kt

```diff
@@ -294,20 +282,20 @@ class ScaleBarImpl : ScaleBar, View {
         )
       }
       val distanceText = getDistanceText(unitDistance * pair.second)
-      textPaint.getTextPath(
-        distanceText, 0, distanceText.length, unitBarWidth * pair.second,
-        textSize, path
-      )
-      if (showTextBorder) {
-        canvas.drawPath(path, strokePaint)
-      }
-      canvas.drawPath(path, textPaint)
+      drawText(canvas, distanceText, unitBarWidth * pair.second, textSize)
     }
     if (useContinuousRendering) {
       reusableCanvas = canvas
     }
   }
 
+  private fun drawText(canvas: Canvas, text: String, x: Float, y: Float) {
+    if (settings.showTextBorder) {
+      canvas.drawText(text, x, y, strokePaint)
+    }
+    canvas.drawText(text, x, y, textPaint)
```

Comment:
shouldn't it be `else` branch?
if `settings.showTextBorder == true` we will draw text twice

Reply:
That's on purpose, please see the attached screenshot, texts have a white border which is drawn by `strokePaint` and the black text is drawn by `textPaint`.

Reply:
was just making sure, thanks.